converter/service/converter.py

```diff
@@ -109,12 +115,14 @@ def handle(decoded_url, user_agent, accept):
     try:
         res = urlopen(req, timeout=MAX_TIMEOUT)
```

Comment:
This returns a context manager so it needs to be closed: https://docs.python.org/3.6/glossary.html#term-context-manager

You can do this with:
```python
with urlopen(...) as res:
  ...
```